scripts/release.js

```diff
@@ -0,0 +1,162 @@
+#!/bin/env node
```

Comment:
Maybe add a short explanation of what this scripts is used for. 
Add a link to public documentation.